### DISCUSSION TITLE
Noob questions in relation to and getting started with your addon

### DISCUSSION THREAD

**Discussion by BeastlordGG** [zealvurte/SimSimMissionWin#13] [Q&A] [answered]
Hi
Does your version work just like VenturePlanSoDMissions in relation to and get addon to and work e.g. " 4.16a. Then open up _retail_ / Interface / AddOns / VenturePlan / vs.lua in a text editor, and insert the line _G [_] = T " or has reached another one to do for and make it work PS I know well i'm a super noob in this area sorry for that
Regards Beastlord

**Comment by zealvurte** [comment, marked as answer]
There is currently no addon for use, and this project's purpose is not to get VP updated and working for others.

I currently provide instructions, a spreadsheet, and a script to generate mostly verified spell data, which can optionally be used for VP or VenturePlanSoDMissions if you're already familiar with Lua and their code. As you've self identified as "a super noob", it sounds like this project won't be helpful to you.
